python/tvm/relay/frontend/mxnet.py

```diff
@@ -1075,6 +1098,377 @@ def _mx_cond(inputs, attrs, subgraphs):
     return ret
 
 
+def _qnn_mx_contrib_quantize(inputs, attrs):
```

Comment:
Can we put functions with `_qnn_mx` prefix into mxnet_qnn_op_utils.py file as well?

Reply:
Renamed the ops so that they no longer have _mx_ prefix in them.